WcaOnRails/app/views/wca_monthly_digest_mailer/send_weat_digest_content.html.erb

```diff
@@ -0,0 +1,11 @@
+<p>Dear WCA Executive Assistants Team,</p>
+
+<p>Here's the draft for this month digest (Please verify whether all the changes were informed to the WCA Board):</p>
+
+<% @teams = Team.unscoped.all %>
+
+<div><b>Changes in Teams/Committees/Councils</b></div>
+
+<% @teams.each do |team| %>
+  <% team.changes_in_team %>
```

Comment:
Seems like nitpick but is actually super important: You have to use `<%=` instead of `<%` to actually _print_ the result of the Ruby code into the mail

Reply:
On that note, can you add some backup code that writes "there were no changes" if the return string is empty?

Reply:
Done, thanks for this.